addons/sale/static/src/js/sale_product_field.js

```diff
@@ -71,6 +71,13 @@ export class SaleOrderLineProductField extends Many2OneField {
         return 'fa-pencil';
     }
 
+    onClick(ev) {
```

Comment:
a docstring to explain why we change the standard behavior would be nice ;).

Reply:
:duck:

Reply:
Also, shouldn't this call super if we do not enter the if ? (probably won't have any impact, but more correct 'architecturalement parlant'